exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,91 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [3, 5, 2, 7]
+      },
+      "expected": [3, 5, 2, 7]
```

Comment:
These scores seem low for 'high' scores.
Perhaps you could help this by just putting a line in the readme that says something like "Scores are usually in millions but we're leaving that bit off." or "It's really hard to get Frogs across the road so 7 is a good score!"

Reply:
I had a good laugh with "It's really hard to get Frogs across the road so 7 is a good score!" 😄 However the exercise only refers Frogger in the `source` bit, which is relatively hidden.

Reply:
> I'd rather not add it to the Readme.

That's fine :heart:
(It is also fine if you don't want any of the "color" I was suggesting in another comment and and just want to retain a short to the point readme.)